tools/public_api_guard/material/autocomplete.md

```diff
@@ -190,7 +190,7 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, AfterViewIn
     // (undocumented)
     _handleKeydown(event: KeyboardEvent): void;
     // (undocumented)
-    static ngAcceptInputType_autocompleteDisabled: unknown;
+    static ngAcceptInputType_autocompleteDisabled: unknown /** Handles keyboard events coming from the overlay panel. */;
```

Comment:
Note that this looks like a bug with the API generation tool. I'm approving it for now so the test gets fixed.

Reply:
👍 Would be Microsoft's API extractor, or maybe ngtsc is doing something surprising that causes this now?

Reply:
I'm guessing it's the API extractor. It's been logging warnings that it was written for TS 5.0 while we're running it with 5.2. It might be time to update it.